src/views/dashboard/DashboardGoogle.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from "react";
+import DarkMode from "../_shared/DarkMode";
+import "../home/Home";
+import { useMutation } from "@apollo/react-hooks";
+//import User from "./User";
+import Button from "@material-ui/core/Button";
+import { UPDATE_USER } from "../../graphql-requests/mutations";
+import TextField from "@material-ui/core/TextField";
+import AddButton from "./AddButton";
+
+const DashboardGoogle: React.FC = (props: any) => {
+   const [name, setName] = useState("");
+
+   useEffect(() => {
+      let username = localStorage.getItem("username");
+      if (username) {
+         setName(username);
+      }
+   }, []);
+
+   const [users, setUsers] = useState({
+      username: "",
+      email: "",
+      first_name: "",
+      last_name: "",
+   });
+
+   const handleChange = (event: any) => {
+      // event.preventDefault();
+      setUsers({
+         ...users,
+         [event.target.name]: event.target.value,
+      });
+   };
+
+   {
+      const [updateUser] = useMutation(UPDATE_USER);
+      let user_id = localStorage.getItem("user_id");
+      const onSubmit = ({
+         user_id = localStorage.getItem("user_id"),
+         first_name,
+         last_name,
+         username,
+         email,
+      }: any) => {
+         console.log(user_id);
+         updateUser({
+            variables: { user_id, ...users },
+         })
+            .then(res => {
+               console.log(res.data);
+               localStorage.setItem("username", res.data.updateUser.username);
+               props.history.push("/homeg");
+               console.log("Successfully updated.");
+            })
+            .catch(err => {
+               alert(err.message);
+               console.log(err);
+            });
+      };
+      return (
+         <>
+            <main className="dashboard">
+               <br />
```

Comment:
Why are we still using <br /> tags instead of allowing the CSS to handle this styling?